### DISCUSSION TITLE
React version

### DISCUSSION THREAD

**Discussion by pcpatfire** [JedWatson/react-select#5128] [Q&A]
I'm trying to use the current version of react-select in my project with react version 17.0.2, but even with the simplest example from your documentation:

```
import React from 'react'

import Select from 'react-select'

const options = [
  { value: 'chocolate', label: 'Chocolate' },
  { value: 'strawberry', label: 'Strawberry' },
  { value: 'vanilla', label: 'Vanilla' }
]

const CustomSelect = () => (
  <Select options={options} />
)

export default CustomSelect;
```
I get the following error:

`Invalid hook call. Hooks can only be called inside of the body of a function component. `

Is there a compatibility problem with my react version? What am I doing wrong?

**Comment by manjushsh** [comment]
Hi @pcpatfire , I tested this code on codesandbox.. It is working just fine
![Screenshot](https://user-images.githubusercontent.com/94426452/160752987-bd9f0b2f-e953-46a6-8f76-1c2190d04574.png)

How are you using `CustomSelect`?
.
